### DISCUSSION TITLE
Parent vs child report psychopathology in ABCD

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#14] [Where in the World is…? (Help)]
Thoughts on using parent report vs child-report psychopathology in ABCD? Is there a right and wrong way to go for a longitudinal analysis?

**Comment by sanaashrafali** [comment]
Paper resources: 

1. [Brain structural associations with depression in a large early adolescent sample (the ABCD study®)](https://www.sciencedirect.com/science/article/pii/S2589537021004855)
2. [Agreement between youth-reported and parent-reported psychopathology in a referred sample | SpringerLink](https://link.springer.com/article/10.1007/s00787-008-0710-z)
3. [https://acamh.onlinelibrary.wiley.com/doi/abs/10.1111/j.1469-7610.1992.tb00922.x?casa_token[…]MBabHCl5Rufjf0X2-qBT1VY-uDtZdpjEUK4nHr3P6PTOAWu-wkPCFA](https://acamh.onlinelibrary.wiley.com/doi/abs/10.1111/j.1469-7610.1992.tb00922.x?casa_token=3jfibRFDxvEAAAAA:-gDDxvFKIQZ8onm-VJPHsadw_0wjMBabHCl5Rufjf0X2-qBT1VY-uDtZdpjEUK4nHr3P6PTOAWu-wkPCFA)
4. [Neighbourhood socioeconomic disadvantage and behavioural problems from late childhood into early adolescence](https://jech.bmj.com/content/57/9/699.short)

- Other suggestion: Looking at multi-trait multi-informant (or multi-method) models.
- Sensitivity analyses would be a suggestion too (possibly pre registered) if you can't use a latent trait model.
- Important note: it might be worth being careful about making decisions based on analyses of ABCD data (to keep analytic decisions independent of the data you're doing the analysis on).

**Comment by stapert** [comment]
Generally, internalizing disorders (mood, anxiety) are best reported by the respondant (the youth), while ADHD symptoms are best reported by others who can observe them (the parent or teacher).
